packages/vite/src/rsc/rscVitePlugins.ts

```diff
@@ -160,6 +133,7 @@ export function rscAnalyzePlugin(
     transform(code, id) {
       const ext = path.extname(id)
       if (['.ts', '.tsx', '.js', '.jsx'].includes(ext)) {
+        // @MARK: We're using swc here, that's cool but another dependency!
```

Comment:
Umm, we can. These MARKS are mainly comments for me (or anyone working on RSC) - because they highlight something unusual/unexpected. It's a good way to improve your understanding of these functions.

I don't think we need to change it just yet though.